src/views/Send/Send.vue

```diff
@@ -293,7 +306,7 @@ export default {
     },
     amountFiat: {
       get () {
-        return `$${this.stateAmountFiat}`
+        return this.stateAmountFiat
       },
       set (newValue) {
         const value = (newValue || '0').replace('$', '')
```

Comment:
I think that we don't need the replace if we are using input type number and you removed the $ sign from the getter